packages/common/src/types/models.ts

```diff
@@ -27,8 +28,8 @@ export type Role = {
 };
 
 export type Booking = {
-  thingId: string;
-  userId: string;
+  thing: { name: string; description: string };
+  user: { fullname: string; id: string };
```

Comment:
```suggestion
  user: { fullName: string; id: string };
```
be careful of the letter case